app/models/service.rb

```diff
@@ -519,6 +519,10 @@ def plugin_deployment?
     DeploymentOption.plugins.include?(deployment_option)
   end
 
+  def top_level_metrics
```

Comment:
Maybe this could move closer to https://github.com/3scale/porta/blob/16e746c7665d179b07f2a5b9415bbb4c6adc92c3/app/models/service.rb#L319